Tools/build/stable_abi.py

```diff
@@ -522,7 +522,7 @@ def gcc_get_limited_api_macros(headers):
 
     api_hexversion = sys.version_info.major << 24 | sys.version_info.minor << 16
 
-    preprocesor_output_with_macros = subprocess.check_output(
```

Comment:
Not sure who the current maintainer here would be (@encukou?) but someone should check that all references have been handled.

Reply:
Yeah, me. I won't get to reviewing a typo in an internal variable name any time soon though. Generally we avoid such cosmetic changes, anyway.